CHANGELOG.md

```diff
@@ -1,8 +1,11 @@
 # Changelog
 
 ## Draft
+- Fix ItemAvailability microdata schema for product pages. [#3168](https://github.com/bigcommerce/cornerstone/pull/1174)
 
 ## 1.13.2 (2018-02-28)
+- Fix updateView firing when there are no default options. [#1172](https://github.com/bigcommerce/cornerstone/pull/1172)
+- Don't clear bulk pricing rules when feature is disabled. [#1173](https://github.com/bigcommerce/cornerstone/pull/1173)
```

Comment:
I am not sure I get what are these changelog entries for ? Did we miss it when we did 1.3.2 ?

Reply:
File it as a separate PR please.

Reply:
@junedkazi 👍